README.md

```diff
@@ -15,6 +15,16 @@ Because we produce multi-platform code and a multi-platform runtime, the code wr
 
 The project should be considered experimental. Several parsers have been implemented and work so far using this target but not all the features are complete and well tested.
 
+The runtime and target's template are up-to-date with mainstream commit: `master/e9df464`  
```

Comment:
@lppedd , could you also add a section about contributors specifying your name. You are showing quite a lot of love to this project and your contribution is making the difference, so I think your name should appear in the README

Reply:
Done!